interface/utils.py

```diff
@@ -57,3 +47,30 @@ def random_code(length, vocabulary=vocabulary_64):
 def is_true(value):
     text = (value or '').lower().strip()
     return text in ['1', 'yes', 'true', 'on', 'enabled']
+
+    
+def get_url_base(link):
+    m = re.match(r'https://github.com/(?P<org>[^/]+)/(?P<repo>[^/]+)/?$',
+                    link.assignment.repo_url)
+    url_base = ('https://raw.githubusercontent.com/'
+                '{0}/{1}/'.format(*list(m.groups())))
+    return url_base
+
+def get_config_url(link):
+    url_base = get_url_base(link)
+    config_url = urljoin(url_base,
+                             f'{link.assignment.repo_branch}/checker.sh')
+    return config_url
```

Comment:
Same thing about the coding style as above applies here. Also, `script_url` is more intuitive than `config_url`.